### ISSUE TITLE
Unblock send

### ISSUE THREAD

**Issue by Parsh** [bithyve/hexa#2675]
We shouldn't block a user from sending if the low priority fee + amount to send > balance. User should be able to send w/ however low fee(till it's greater than or equal to the minimum fee to relay).

Connected to #2657 (part 2)

**Comment by cakesoft-swati** [comment]
-Verified this issue on staging app v1.4.5(227)
